examples/qml_features/rust/src/lib.rs

```diff
@@ -16,6 +16,7 @@ pub mod custom_parent_class;
 pub mod externcxxqt;
 pub mod invokables;
 pub mod multiple_qobjects;
+mod naming;
```

Comment:
could add `pub` here then you'll need to add docs to things, but at least ensures it works with the renamed things :-)